Add support for using a monochrome icon for Menu Bar Workspaces

mads-hartmann opened the pull request:
This PR makes it possible to use a monochrome icon for Menu Bar Workspaces. It also uses the monochrome icon as the default to blend in more with the standard MacOS experience. 

A few screenshots:

![Screenshot 2023-11-23 at 11 49 08](https://github.com/gitpod-samples/Gitpod-Raycast-Extension/assets/83561/26f44e74-6272-45c8-86ee-4b401d8dbd35)
![Screenshot 2023-11-23 at 11 49 14](https://github.com/gitpod-samples/Gitpod-Raycast-Extension/assets/83561/47e15c78-b854-4a6e-83a6-17d4a6730538)

mads-hartmann commented:
@Palanikannan1437 and @henit-chobisa I added support for configuring the icon style to preserve the orange color for people who prefer that. But I wanted to ask how you feel about not having it be configurable. The GitHub and Linear extensions don't make it configurable and they use monochrome by default ☺️

Palanikannan1437 commented:
> @Palanikannan1437 and @henit-chobisa I added support for configuring the icon style to preserve the orange color for people who prefer that. But I wanted to ask how you feel about not having it be configurable. The GitHub and Linear extensions don't make it configurable and they use monochrome by default ☺️

I always liked the Gitpod orange differentiator in my menu bar, but for many it makes sense to go MacOS standards ie monochrome. So best choice moving forward seems to be keeping it configurable, thank you so much for this and taking the time for putting thoughts into this ✌️

Also, the GitHub notifications menubar Raycast extension command actually turns Golden if you have unread notifications....so could think of something in that direction too 🙂

mads-hartmann commented:
@Palanikannan1437 Awesome, I moved the PR out of draft so feel free to review it. I also added screenshots showing what it looks like in monochrome ☺️ @gtsiolis opened https://github.com/gitpod-samples/Gitpod-Raycast-Extension/pull/74 which should fix some padding with the icon but the PRs can be merged/review independently.

mads-hartmann commented:
@Palanikannan1437 Okay, turns out it won't switch colours today when I "Toggle System Appearance" so moving back to draft 😝 Will see if I can figure out what's going on later
